src/SwiftUI/SwiftUI/Color.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	using static Color;
+
+	public enum RGBColorSpace
+	{
+		DisplayP3,
+		RGB,
+		RGBLinear,
+	};
+
+	[SwiftImport (SwiftUILib.Path)]
+	public unsafe partial class Color : View
+	{
+		public IntPtr Data { get; private set; }
```

Comment:
This should not be public.
```suggestion
		readonly IntPtr data;
```